style.css

```diff
@@ -316,4 +316,96 @@
     line-height: 20px; /* 133.333% */
     padding-top: 15px;
   }
+
+  .contact-form {
+    background-color: #fff;
+  }
+
+  .bg-container-contact {
+    background-color: #6070ff;
+    border-top-left-radius: 4em;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 60px 20px 40px 20px;
+  }
+
+  .contact-heading {
+    color: var(--neutrals-n-0, #fff);
+    text-align: center;
+    font-family: Poppins, Arial, Helvetica, sans-serif;
+    font-size: 40px;
+    font-style: normal;
+    font-weight: 700;
+    margin-bottom: 10px;
+    padding-bottom: 0;
+  }
+
+  .contact-description {
+    color: var(--primary-b-50, #ebebff);
+    text-align: center;
+    font-family: Poppins, Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    font-style: normal;
+    font-weight: 400;
+    margin-bottom: 15px;
+    line-height: 28px; /* 140% */
+  }
+
+  .unordered-contact-form {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    padding-left: 25px;
+    gap: 14px;
+  }
+
+  input {
+    border-radius: 8px;
+    border: 1px solid #cfd8dc;
+    background: #fff;
+    width: 327px;
+    height: 48px;
+    flex-shrink: 0;
+    padding: 10px;
+    color: var(--neutral-n-800, #172b4d);
+    font-family: Poppins, Arial, Helvetica, sans-serif;
+    font-size: 17px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 20px; /* 117.647% */
+  }
+
+  textarea {
+    width: 327px;
+    height: 140px;
+    flex-shrink: 0;
+    border-radius: 8px;
+    border: 1px solid #cfd8dc;
+    background: #fff;
+    padding: 10px;
+  }
+
+  .contact-button {
+    border-radius: 8px;
+    background: var(--neutrals-n-0, #fff);
+    padding: 12px 16px;
+    align-items: center;
+    display: inline-flex;
+    color: var(--primary-b-400, #6070ff);
+    text-align: center;
+    font-family: Poppins, Arial, Helvetica, sans-serif;
+    font-size: 17px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 24px; /* 141.176% */
+    letter-spacing: 0.51px;
+    border: 0 solid #fff;
+  }
```

Comment:
- [ ] You did well styling the form but the submit button on hover disappears, you should use the same button interactions such as hover, active (when it's pressed), disabled and enable (default state) as shown in the Figma template.


<img width="607" alt="Screenshot 2023-07-21 at 12 37 27 AM" src="https://github.com/PremChand12/mobile-first-project/assets/53347966/7a5714a4-e218-492a-91c7-7e145dc73a0a">